data/json/recipes/recipe_food.json

```diff
@@ -59,6 +59,44 @@
   ]
 },{
   "type" : "recipe",
+  "result": "salt",
+  "id_suffix": "from_mineral",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "cooking",
+  "difficulty": 4,
+  "skills_required": [ "cooking", 4 ],
+  "time": 90000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [["atomic_survival", 4], ["textbook_chemistry", 5], ["adv_chemistry", 4]],
+  "batch_time_factors":[80, 4],
+  "qualities":[
+    {"id":"BOIL","level":1,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 3 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 3 ],
+      [ "coffeemaker", 3],
+      [ "atomic_coffeepot", -1 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "material_rocksalt", 2 ]
```

Comment:
This looks like 2 charges of rock salt per 8 charges of salt. Since rock salt stacks in charges of 50, this would make rock salt 12.5 times more volume efficient than pure salt.